Jenkins_jobs/WorkspaceInfo.groovy

```diff
@@ -1,138 +1,100 @@
 @Library('NodeHelper') _
 import jenkins.model.Jenkins;
 import hudson.model.Computer;
+import org.jenkinsci.plugins.workflow.steps.TimeoutStep;
 
 clones = [:]
-List<String> newMachines = new ArrayList<String>();
 
 /* Iterates over all the online nodes in Jenkins and
  * prints contents of workspace folder along with
  * the space they occupy. With exception of tmp directory
```

Comment:
I don't understand the comment about the tmp dir

Reply:
tmp directory is any directory with "@tmp" in the name. I'll update the comment